src/pymovements/plotting/traceplot.py

```diff
@@ -37,44 +42,59 @@
 if 'pytest' in sys.modules:  # pragma: no cover
     matplotlib.use('Agg')
 
-DEFAULT_SEGMENTDATA = {
+LinearSegmentedColormapType: TypeAlias = Dict[
+    Literal['red', 'green', 'blue', 'alpha'], Sequence[Tuple[float, ...]],
```

Comment:
can you use `tuple` instead of `Tuple` here?

Reply:
no unfortunately only works for 3.9+ and we (still) support 3.8